getting-started/installation-macports.md

```diff
@@ -0,0 +1,55 @@
+## MacPorts [](macports)
+
+This will install both TimescaleDB *and* PostgreSQL via MacPorts.
+
+**Note: TimescaleDB requires PostgreSQL 11 or 12.**
+
+#### Prerequisites
+
+- [MacPorts][]
+
+#### Build & Install
+
+>:WARNING: If you have another PostgreSQL installation
+(such as through Postgres.app), the following instructions may
+cause problems. If you wish to maintain your current version of PostgreSQL
+outside of MacPorts we recommend installing from source.
+
+```bash
+# To install
+port -v install timescaledb +postgresql:pg_version:
+
+# Post-install to move files to appropriate place
+/usr/local/bin/timescaledb_move.sh
```

Comment:
is this part needed for Macports?

Reply:
Not sure

Reply:
No, the `timescaledb_move.sh` isn't needed. After running the `port install ...` then I can immediately use timescaledb.